pymobiledevice3/cli/developer.py

```diff
@@ -271,6 +277,32 @@ def screenshot(service_provider: LockdownClient, out):
         out.write(Screenshot(dvt).get_screenshot())
 
 
+def app_lookup(install_service: InstallationProxyService, bundle_id: str):
+    # install_service.lookup is not working for RSD，use browse instead
```

Comment:
please prefix this with a `TODO:` so it's easier to get back to it.
also, that's a bit wierd. please create an issue for it since it should work the same.

Reply:
https://github.com/doronz88/pymobiledevice3/issues/779